README.md

```diff
@@ -118,6 +118,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |2|Generic connectors sample | Sample code for a generic connector that's easy to customize for any system which supports webhooks.   |    |[View][connector-generic#ts]
 |3|Outgoing Webhooks      | Samples to create "Custom Bots" to be used in Microsoft Teams.                                        |[View][outgoing-webhook#cs]|[View][outgoing-webhook#ts]
 |4|Authentication in todo Connector App      | This is a sample app which shows connector authentication and sends notification on task creation.                                        ||[View][auth-in-connector#nodejs] 
+|5|Incoming webhook|This sample illustrates usage of incoming webhook.||[View][incoming-webhook#nodejs]
```

Comment:
Here in description we should describe more about the use case, like, we used editor and then webhook action part.

Reply:
updated description